src/components/Home/Home.js

```diff
@@ -7,6 +7,12 @@ import CortexCardContainer from './CortexCardContainer/CortexCardContainer';
 import CortexCardContents from './CortexCardContents/CortexCardContents';
 import MagicArrow from './MagicArrow/MagicArrow';
 
+/*i pray to god (if there is one) that there is a beter way to go about this than just manually importing every images*/
```

Comment:
Kill off this comment please.
For your future knowledge, a good way to do this sort of thing is leave your own comment on the PR, explaining that you're unsure of whether this is the right way to do something. It means cleaner code and it's pretty much impossible for reviewers to miss.